app/components/ranked-stats/ranked-stats.html

```diff
@@ -1,12 +1,33 @@
 <h3>Summoner Name: {{summonerName}}</h3>
 
 
+<table class="table">
+    <thead class="match-info-thead">
+    <tr>
 
-<div ng-repeat = "champion in champions">
-    Champion Id: {{champion.id}}
-    Record: {{champion.stats.totalSessionsWon}} - {{champion.stats.totalSessionsLost}}
-    Kills: {{champion.stats.totalChampionKills}}
-    Assists: {{champion.stats.totalAssists}}
-    Deaths: {{champion.stats.totalDeathsPerSession}}
-    Win percentage: {{champion.stats.winPercentage}}
-</div>
+        <th ng-click="orderProp='-id'">Champion</th>
+        <th ng-click="orderProp='-id'">Champ ID</th>
+        <th ng-click="orderProp='-stats.totalSessionsPlayed'">GP</th>
+        <th ng-click="orderProp='-stats.winPercentage'">Win PCT</th>
+        <th ng-click="orderProp='-stats.totalChampionKills'">Kills</th>
```

Comment:
How about instead of kills deaths and assists we give out average kills deaths and assists per game.